test/jdk/java/nio/charset/Charset/AvailableCharsetNames.java

```diff
@@ -22,30 +22,24 @@
  */
 
 /* @test
- * @bug 4422044
+ * @bug 4422044 8310049
```

Comment:
As this is just a refactor, I don't think you need the bug number for converting to junit

Reply:
Fixed, in this file and the others